core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -10,57 +10,90 @@
 // It is not updated automatically.
```

Comment:
BTW, we should add a copyright header at the top of this file.

Reply:
this would be a great task for an automated scanner.